### ISSUE TITLE
abnormal cardiac muscle cell calcium level

### ISSUE THREAD

**Issue by slaulederkind** [mgijax/mammalian-phenotype-ontology#3601]
>If this is a request for a new term, please provide us with the following information.

>1. Preferred term label (e.g., asplenia)

a.  abnormal cardiac muscle cell calcium level
b.  increased cardiac muscle cell calcium level
c.  decreased cardiac muscle cell calcium level

>2. Synonyms (e.g., absent spleen)

a.  abnormal cardiomyocyte calcium level; abnormal cardiac myocyte calcium level;  abnormal heart muscle cell calcium level
b.  increased cardiomyocyte calcium level; increased cardiac myocyte calcium level;  increased heart muscle cell calcium level
c.  decreased cardiomyocyte calcium level; decreased cardiac myocyte calcium level;  decreased heart muscle cell calcium level;  reduced cardiomyocyte calcium level; reduced cardiac myocyte calcium level;  reduced heart muscle cell calcium level

>3. Textual definition (the definition should be understandable even for non-specialists. Include a PubMed ID or URL(s) to refer to any relevant article(s) that provides information about the suggested term).

a. anomaly in the amount of calcium in cardiomyocytes
b. increase in the amount of calcium in cardiomyocytes
c. decrease in the amount of calcium in cardiomyocytes

PMID:32207065

>4. Any additional notes to clarify the meaning or use of the term. Please provide a PubMed ID or URL(s) for the source of this information.

>5. Suggested parent term (Please look in the hierarchy of the MP using the [MP Browser](http://www.informatics.jax.org/vocab/mp_ontology) or other browsers such as [OLS](http://www.ebi.ac.uk/ols/ontologies/mp) and find the place in the MP where the new term best fits)

a. abnormal calcium level (MP:0020879)
b. abnormal cardiac muscle cell calcium level
c. abnormal cardiac muscle cell calcium level

>6. If you would like a nanoattribution, please indicate your ORCID id or provide a label such as MGI:sbello (organization, first initial-last name)

ORCID:0000-0001-5256-8683

**Comment by sbello** [comment]
abnormal - MP:0021139
increased - MP:0021140
decreased - MP:0021141
